src/Configuration.php

```diff
@@ -35,12 +35,32 @@ final class Configuration
     /** @var bool */
     private $flushEnabled = true;
 
-    public function __construct()
+    /** @var bool */
+    private $databaseResetEnabled = true;
+
+    /** @var list<string> */
+    private $ormConnectionsToReset;
+
+    /** @var list<string> */
+    private $ormObjectManagersToReset;
+
+    /** @var string */
+    private $ormResetMode;
+
+    /** @var list<string> */
+    private $odmObjectManagersToReset;
```

Comment:
I'm wondering if this is the right place for these options...
here are few other options:
- declare a `DatabaseResetterConfiguration` service
- transform `DatabaseResetter` into a service, and it will be injected directly all these stuff
- transform `DatabaseResetter` and `ORMDatabaseResetter` and `ODMSchemaResetter` into a service, and each one will be injected with its related options

Reply:
Let's leave here, the class is internal so we can move later w/o consequence.